### PR TITLE
Import excel

### PR DESCRIPTION
**Pull request by romfabbro**

### CODE REVIEW

File: Front/app/modules/importFile/excel/lyt-step2-excel.js [hunk 1]
```diff
@@ -59,60 +57,76 @@ define([
...
         });
     },
 
     startWebSocket: function(guid){
       var _this = this;
-         var ws = new WebSocket("ws://127.0.0.1:6545/ecoReleve-Websockets/fileImport/"+guid);
-        ws.onmessage = function(msg) {
-          _this.$el.find('#js-wsmsg').append("<span>" + msg.data + "</span></br>");
+      // domain name should be in config
+      var ws = new WebSocket('ws://127.0.0.1:6545/ecoReleve-Websockets/fileImport/' + guid);
```
Comment (Matheo13) [inline Front/app/modules/importFile/excel/lyt-step2-excel.js]: If u use websockets, domain & port should be in config file.

**Comment by Matheo13** [comment]
why?

**Comment by jvitus** [comment]
mega failed 
 git push romain :import_excel
To https://github.com/romfabbro/ecoReleve-Data.git
 - [deleted]           import_excel 

....

**Comment by romfabbro** [comment]
to REBOOT with less commit

**Pull request closed without merging**